src/System.Linq.Expressions/tests/InterpreterTests.cs

```diff
@@ -17,7 +17,8 @@ public static class InterpreterTests
         [Fact]
         public static void VerifyInstructions_Simple()
         {
-            Expression<Func<string, bool>> f = s => s != null && s.Substring(1).Length * 2 > 0;
+            // Using an unchecked multiplication to ensure that a mul instruction is emitted (and not mul.ovf)
+            Expression<Func<string, bool>> f = s => s != null && unchecked(s.Substring(1).Length * 2) > 0;
```

Comment:
I'm not sure whether this comment is necessary, but as it's part of an expression it would be `Multiply` vs `MultiplyChecked` rather than `mul` vs `mul.ovf`.